containers/dashboard/id/modals/todoCreateModal/TodoCreateModal.tsx

```diff
@@ -0,0 +1,70 @@
+import styles from './TodoCreateModal.module.scss';
+import Image from 'next/image';
+import putImg from '@/assets/images/img_todoSample.png';
+export default function TodoCreateModal() {
+  return (
+    <div className={styles['container']}>
+      <form className={styles['form']}>
+        <p className={styles['modal-title']}>할 일 생성</p>
+        <div className={styles['owner']}>
+          <div className={styles['label-and-form']}>
+            <label className={styles['form-label']}>담당자</label>
+            <select className={styles['dropdown-preview']}>
+              <option>장아영</option>
+              <option>최민경</option>
+            </select>
+          </div>
+        </div>
+        <div className={styles['label-and-form']}>
+          <div className={styles['label-with-star']}>
+            <label className={styles['form-label']}>제목</label>
+            <label className={styles['essential']}>*</label>
+          </div>
```

Comment:
라벨에 htmlFor 속성 주도록 수정하실텐데 그러면 속성을 두 번 주셔야 할거 같아요!
```suggestion
<label htmlFor='' className={styles['form-label']}>
      제목 <span className={styles['essential']}>*</span>
</label>
```
span으로 처리해보시는건 어떠세용?